wonnx-preprocessing/src/text.rs

```diff
@@ -178,6 +128,75 @@ impl BertEncodedText {
             .map(|x| *x as i64)
             .collect()
     }
+
+    pub fn get_answer(&self, start_output: &[f32], end_output: &[f32], context: &str) -> Answer {
+        let mut best_start_logit = f32::MIN;
+        let mut best_start_idx: usize = 0;
+
+        let input_tokens = self.encoding.get_tokens();
+
+        for (start_idx, start_logit) in start_output.iter().enumerate() {
+            if start_idx > input_tokens.len() - 1 {
+                break;
+            }
+            match input_tokens[start_idx].as_str() {
+                "[CLS]" | "[SEP]" | "[PAD]" => continue,
```

Comment:
Could those tokens be constant variables?

Reply:
Good idea, I now use the `special_tokens_mask` from the tokenizer to skip special tokens (which means it will skip any special token configured for the tokenizer in e.g. the JSON).